src/backend/assets/js/social.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+$('.social-button').click(function() {
+  var source = this.id.toString().split('-')[0];
+
+  if (source === 'fb') {
+    window.open('http://www.facebook.com/share.php?u=' + window.location, 'facebook', 'height=360, width=580');
+  } else if (source === 'tw') {
+    window.open('http://twitter.com/home?status=' + window.location, 'twitter', 'height=360, width=580');
+  } else if (source === 'go') {
+    window.open('https://plus.google.com/share?url=' + encodeURIComponent(window.location), 'google', 'height=360, width=580');
```

Comment:
After testing the build on heruku, the url without `encodeURIComponent ` was invalid url for gplus, after adding `encodeURIComponent ` the url was no more invalid.